### PR TITLE
feat: Add Getting Started page

### LINKED ISSUE #435: Add Getting Started Page for OpenShift Toolkit
[kind/Feature]

## Which functionality do you think we should add?

Add Getting Started/Walkthrough for OpenShift workflow. These will be opened when the extension is loaded and can be replicated on the lines of the VSCode OpenShift extension.

## Why is this needed?

This will allow users to follow a set of steps to work with the extension in a guided way.

### LINKED ISSUE #479: Debug menu item is not shown although debug is available for component
[kind/bug]

## What did you run exactly?

Create a Quarkus or Node.js component from registry

## Actual behavior

Debug menu item is not shown

## Expected behavior

Debug menu item should be shown

### PR DESCRIPTION
**Pull request by jeffmaury**
Fixes #435
Fixes #479

Signed-off-by: Jeff MAURY <jmaury@redhat.com>

## Was the change discussed in an issue?
fixes #???

### CODE REVIEW

File: src/main/java/org/jboss/tools/intellij/openshift/GettingStartedToolWindow.java [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+                "various resources (URL, Storage, Services, etc.) that correspond to your component</p></html>",
+                Collections.emptyList(),
+                gifCreateComponentLesson
+        );
+
+        URL gifStartDevLesson = getLessonGif("start-dev.gif");
+        GettingStartedLesson startDevLesson = new GettingStartedLesson(
+                "Start a component in development mode",
+                "<html><p>This is inner loop development and allows you to code, build, run and test the" +
+                        "application in a continuous workflow. It continuously watches teh directory for any new " +
```
Comment (lstocchi) [inline src/main/java/org/jboss/tools/intellij/openshift/GettingStartedToolWindow.java:121]: typo `teh`

File: src/main/java/org/jboss/tools/intellij/openshift/GettingStartedToolWindow.java [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+                        "<p>This also forwards a port on the development system to the port on the container cluster " +
+                        "allowing you remote access to your deployed application</p></html>",
+                Collections.emptyList(),
+                gifStartDevLesson
+        );
+
+        URL gifStartDebugLesson = getLessonGif("start-debug.gif");
+        GettingStartedLesson startDebugLesson = new GettingStartedLesson(
+                "Debug the component",
+                "<html><p>Start the component in debug mode. The devfile.yaml should be specified with the" +
```
Comment (lstocchi) [inline src/main/java/org/jboss/tools/intellij/openshift/GettingStartedToolWindow.java:132]: no space at the end of the row so you see `thedebug` in the IDE

**Review by lstocchi — APPROVED** [review]
LGTM, just a typo and missing spaces

File: src/main/java/org/jboss/tools/intellij/openshift/GettingStartedToolWindow.java [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+                "associated 'devfile.yaml' configuration. Devfile is a manifest file that contains information about " +
+                "various resources (URL, Storage, Services, etc.) that correspond to your component</p></html>",
+                Collections.emptyList(),
+                gifCreateComponentLesson
+        );
+
+        URL gifStartDevLesson = getLessonGif("start-dev.gif");
+        GettingStartedLesson startDevLesson = new GettingStartedLesson(
+                "Start a component in development mode",
+                "<html><p>This is inner loop development and allows you to code, build, run and test the" +
```
Comment (sbouchet) [inline src/main/java/org/jboss/tools/intellij/openshift/GettingStartedToolWindow.java]:
```suggestion
                "<html><p>This is inner loop development and allows you to code, build, run and test the  " +
```

File: src/main/java/org/jboss/tools/intellij/openshift/GettingStartedToolWindow.java (hunk 1 above)
Comment (sbouchet) [inline src/main/java/org/jboss/tools/intellij/openshift/GettingStartedToolWindow.java]:
```suggestion
                        "application in a continuous workflow. It continuously watches the directory for any new " +
```

File: src/main/java/org/jboss/tools/intellij/openshift/GettingStartedToolWindow.java (hunk 2 above)
Comment (sbouchet) [inline src/main/java/org/jboss/tools/intellij/openshift/GettingStartedToolWindow.java]:
```suggestion
                "<html><p>Start the component in debug mode. The devfile.yaml should be specified with the " +
```

**Review by sbouchet — CHANGES_REQUESTED** [review]
please fix the typos.
also, the devfile registry gif might look better :
- by removing the top pixels that are disturbing
- by removing/cropping the loading times

File: src/main/java/org/jboss/tools/intellij/openshift/GettingStartedToolWindow.java [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+                "<html><p>Select a specific devfile registry to create a component directly.</p></html>",
+                Collections.emptyList(),
+                gifBrowseDevfileRegistryLesson
+        );
+
+        URL gifCreateComponentLesson = getLessonGif("create-component.gif");
+        GettingStartedLesson createComponentLesson = new GettingStartedLesson(
+                "Create a Component",
+                "<html><p>Create a component in the local workspace using the selected started project and " +
+                "associated 'devfile.yaml' configuration. Devfile is a manifest file that contains information about " +
```
Comment (mohitsuman) [inline src/main/java/org/jboss/tools/intellij/openshift/GettingStartedToolWindow.java:111]:
```suggestion
                "associated 'devfile.yaml' configuration. With devfiles, you record the instructions for configuring and running your build environment as a YAML-formatted text file and provide Build, Run and Deployment information within the file." +
```

File: src/main/java/org/jboss/tools/intellij/openshift/GettingStartedToolWindow.java [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+                "various resources (URL, Storage, Services, etc.) that correspond to your component</p></html>",
```
Comment (mohitsuman) [inline src/main/java/org/jboss/tools/intellij/openshift/GettingStartedToolWindow.java:112]:
```suggestion
                "</p></html>",
```

**Review by mohitsuman — CHANGES_REQUESTED** [review]
Few wording changes.

**Review by mohitsuman — APPROVED** [review]
LGTM

**Pull request merged**